tests/modules/atomicslotmigration.c

```diff
@@ -234,6 +241,33 @@ static void testNonFatalScenarios(RedisModuleCtx *ctx, RedisModuleClusterSlotMig
     testReplicatingUnknownCommand(ctx);
 }
 
+int disableTrimCmd(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    REDISMODULE_NOT_USED(argv);
+    REDISMODULE_NOT_USED(argc);
+    disableTrimFlag = 1;
```

Comment:
So those commands bypass the `cluster_module_trim_disablers` refcount mechanism? Isn't it dangerous?

Reply:
Then maybe add a comment explaining that.